example/mii_core_data_set/generate_cds.py

```diff
@@ -289,6 +289,42 @@ def write_ui_trees_to_files(trees: List[TermEntry], directory: str = "ui-trees")
         write_object_as_json(tree, f"{directory}/{tree.display}.json")
 
 
+def write_used_value_sets_to_files(ui_profiles: List[UIProfile], directory: str = "value-sets"):
+    """
+    Writes the value sets used in the ui profiles to the value-sets folder
+    :param ui_profiles: ui profiles to extract the value sets from
+    :param directory: directory to write the value sets to
+    """
+    value_sets = list()
```

Comment:
this looks like it is writing the value sets not as original value sets from the ontoserver but instead the list of the selectable concepts. 
I would have expected value sets analogous to: https://github.com/medizininformatik-initiative/fhir-ontology-generator/tree/48-add-dse-file-generation/example/merged-ontology/ontology/backend/value-sets